sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -80,7 +80,7 @@
 
 public class DocIndexMetaData {
 
-    private static final String ID = "_id";
+    public static final String ID = "_id";
```

Comment:
Maybe remove this and replace with `DocSysColumns.Names.ID`, so that we have a single source for that.